docs/topics/data-classes.md

```diff
@@ -1,6 +1,6 @@
 [//]: # (title: Data classes)
 
-Data classes in Kotlin are classes whose main purpose is to hold data. Data classes come automatically with additional
+Data classes in Kotlin are primarily used to hold data. Data classes come automatically with additional
```

Comment:
Your edit is fine, but I can't help being amused with the phrase "data classes come automatically" ) it could be improved with an actor, like: "For each data class, compiler automatically generates additional member functions..."

Reply:
Thank you. :) I agree, it does sound a bit weird, and I like this change. I'm implementing it.